src/infra/postgres/postgres.go

```diff
@@ -12,6 +12,7 @@ type Client interface {
 	SelectDeletedPK(ctx context.Context, model ...interface{}) error
 	Select(ctx context.Context, model ...interface{}) error
 	SelectDeleted(ctx context.Context, model ...interface{}) error
+	SelectWhere(ctx context.Context, model interface{}, condition string, params ...interface{}) error
```

Comment:
Minor: I would rather rename `condition` by `where` just for correlation with the method name

Reply:
good idea